File: UVa100.py
while 1:
	try:
		a,b = map(int,input().split())
		ans = -1
		for i in range(min(a,b),max(a,b) + 1):
			n = i
			count = 1
			while n != 1:
				count += 1
				if n % 2:
					n = 3 * n + 1
				else:
					n //= 2
			ans = max(count,ans)
		print(a,b,ans)
	except:
		break

File: UVa100.c
#include<stdio.h>
int main(){
	int a, b;
	while(scanf("%d %d", &a, &b) != EOF){
		printf("%d %d ", a, b);
		if(a > b){
			int t = a;
			a = b;
			b = t;
		}
		int ans = -1;
		for(int i = a;i <= b;i++){
			int n = i,count = 1;
			while(n != 1){
				count++;
				if(n % 2 == 1){
					n = n * 3 + 1;
				}else{
					n = n / 2;
				}
			}
			if(ans < count){
				ans = count;
			}
		}
		printf("%d\n",ans);
	}
	return 0;
}